kalaponents.module

```diff
@@ -82,19 +84,19 @@ function theme_paragraphs_item__kalaponent__full(&$vars) {
 
       // Link text and url as separate variables.
       case 'field_kalaponents_link':
-        $variables['link_text'] = $paragraph->$field->url->value();
-        $variables['link_href'] = $paragraph->$field->title->value();
+        $vars['link_text'] = $paragraph->$field->url->value();
```

Comment:
Good idea here. Actually though I'd rather not send all the Drupal cruft from the `$vars` array into the Twig template. We only want to send the field data.